woocommerce/woocommerce
# Change the imageSizing to single in patterns

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR changes the `imageSizing` attribute for the following patterns since the images need a higher resolution (are currently blurry):
- Product Collection 4 Columns
- Product Collection 3 Columns
- Product Collection 5 Columns
- Product Collection: Featured Products 5 Columns
- Product Collections Featured Collection
- Product Collections Newest Arrivals
- Featured Products 2 Columns
- Product Gallery





Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new page or post.
2. Insert all the patterns mentioned above.
3. Check the product images are not blurry and are using the `single` style.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Update the pattern imageSizing to single.
#### Comment 

</details>
